Mobile middle section project

Alfred-KInyua opened the pull request:
- [ ] build the middle section of the mobile website.
- [ ] used CSS Grid and Flexbox combined. 
- [ ] Used ul and li for categories in "Work" section and "About myself" section.

Alfred-KInyua committed:
Add task files

Alfred-KInyua committed:
Add Index.html File

Alfred-KInyua committed:
Add styles.css file

Alfred-KInyua committed:
Resolve linter errors

Alfred-KInyua committed:
Resolve html error

williamrolando88 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+                </ul>
+            </section>
+        </main>
+    </div>
```
- [ ] Please merge this block of code in your main HTML file, the one you created in the previous exercise. For this portfolio exercise, you only need to create ONE HTML document.

williamrolando88 commented on styles.css:
```diff
@@ -0,0 +1,162 @@
+.container {
+  margin: 0 auto;
+  width: 375px;
+  height: 1263px;
```
- [ ] [Optional] I strongly suggest you delete all fixed width and height properties because in the future is going to be so hard to achieve a responsive design

williamrolando88 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+</head>
+
+<body>
+    <div class="container">
```
- [ ] After you merge this HTML into your main HTML, please change this `div` into a more semantic tag, in this case you must use `section` tag according to your requirements

williamrolando88 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+<body>
+    <div class="container">
+        <main>
+            <p class="paragraph_one">About <br>Myself</p>
```
- [ ] This title has to be wrapped within an `hx` tag.

williamrolando88 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+                    don’t hestiate to contact me.
+                </li>
+                <li class="list_item_two">LET'S CONNECT</li>
+            </ul>
```
- [ ] [Optional] I strongly suggest you not use a `ul` tag for this section, please look at the proper usage of [`ul`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ul).

williamrolando88 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+                    my work and experience! If you like what
+                    you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </li>
```
- [ ] Is required that this block of text be wrapped in a `p` tag. Please make the required changes

williamrolando88 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+
+            <p class="section1"> Languages <img src="images/Union.png" alt="drop down" /></p>
+            <ul class="languages">
+                <li class="l-items"> <img src="images/js.png" alt="java">Javascript</li>
+                <li class="l-items"> <img src="images/html.png" alt="html">HTML</li>
+                <li class="l-items"> <img src="images/CSS.png" alt="css">CSS</li>
```
- [ ] According to your Figma template, please make sure that the text and the icons are properly centered, if you are using flex-box this could be achieved easily by adding `align-items: center`

Your project:
![image](https://user-images.githubusercontent.com/52635378/165316917-ccb2b8f3-d8f2-4979-a021-1d79f99a6677.png) 

Figma template:
 ![image](https://user-images.githubusercontent.com/52635378/165317226-c6e86367-16a4-4078-8e71-db57de3a211b.png)

williamrolando88 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+            </ul>
+            <div class="social-media-icons">
+                <ul class="icons">
+                    <li><img src="images/twitter.png" alt="twitter"></li>
+                    <li><img src="images/linkedin.png" alt="linked in"></li>
+                    <li><img src="images/mid.png" alt="mid"></li>
+                    <li><img src="images/git.svg" alt="git"></li>
+                    <li><img src="images/fin.svg" alt="fing"></li>
```
- [ ] Social icons are hyperlink elements, for them to work please make sure to wrap the `img` tag within an `a` tag

williamrolando88 commented on index.html:
```diff
@@ -0,0 +1,58 @@
+                    <li><img src="images/fin.svg" alt="fing"></li>
+                </ul>
+            </div>
+            <button type="button">Get my resume</button>
```
- [ ] According to your Figma template, this button needs to follow these guidelines:

![image](https://user-images.githubusercontent.com/52635378/165315303-6b6c1481-4666-46b8-a59b-84aff9c7ed2e.png)

williamrolando88 reviewed (changes requested):
Hi @EasyPanther, 

### Good job so far!  👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Your repository looks professional and clean
- Your code is nice and organized
- Good job implementing the Figma designs

## Required Changes ♻️

- [ ] You forgot to add all the required features for the "Works" section, please make sure to add them. This is your [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) use it as guidance
- [ ] Please continue adding this and future requirements for the portfolio project in the same HTML you started working on the last exercise

## Optional suggestions

- [ ] [Optional] I strongly suggest you work on deleting all the fixed values for width and height that you have in your style, when you add a fixed value this means that no matter the size of the screen you cannot resize your webpage, so is not responsive at all. You can achieve all the proper sizing and placement almost all the time, only using flex-box and grid, and padding and margin.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification, also you can send me a Slack message.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Alfred-KInyua commented:
@williamrolando88 
Thanks for the review. 
I am trying to resize flex, can I use the max: width property?

williamrolando88 commented:
> @williamrolando88 Thanks for the review. I am trying to resize flex, can I use the max: width property?

Hi, 
Depending of the case you can use `max-width` or `min-width`, (the same goes for height). Also if you really need to use width or height directly, you can do it with a percentage value instead of a fixed one. You can take a look at this [article](https://www.w3schools.com/cssref/css_units.asp) to learn more about units

Alfred-KInyua commented:
@williamrolando88  
I have learned so much from your reviews, I had to redo the entire project again and I've really enjoyed it (although it was a lot of work). I look forward to sessions like these ones. Thanks a lot.
Let me now push my PR.

Alfred-KInyua committed:
Add image files

Alfred-KInyua committed:
Add html and css files

williamrolando88 commented:
> @williamrolando88  
> I have learned so much from your reviews, I had to redo the entire project again and I've really enjoyed it (although it was a lot of work). I look forward to sessions like these ones. Thanks a lot.
> Let me now push my PR.

If there is anything else you want to ask me, you can always contact me on slack, you can find me with same ID like in github

Alfred-KInyua commented:
@williamrolando88 Thank you sir, I will definitely do that.
please have a look at my project.

m05e5 reviewed (changes requested):
Hi @EasyPanther , 

Good job so far. I really like what you did!
There are some issues that you still need to work on to go to the next project but you are almost there!
![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)


## Highlights


## Required Changes ♻️
* You really did a great job but there are some features you need to improve

- [ ] Please kindly add some styling to your "Get my resume" button
**your design**
![image](https://user-images.githubusercontent.com/37901469/165564294-4aee1b99-4395-4e0d-bd34-4524ed8a8ce9.png)

**Figma design**
![image](https://user-images.githubusercontent.com/37901469/165565306-c74fc59e-bd29-4444-9576-618d9ba50f93.png)


- [ ] As part of the main requirements, please add a **project** section to your portfolio. The project section is an essential part of a portfolio so lets make it look good 😄.
**Figma design**
![image](https://user-images.githubusercontent.com/37901469/165565008-14dd78ef-3190-4bfa-ac35-3ecfd8692cfe.png)

_I'm sure you can do it just keep it up_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_@m05e5_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Alfred-KInyua commented:
@m05e5 
Hello sir, thanks for the review,
With regards to the project section. should I add it as a button just below my portfolio?

Alfred-KInyua committed:
Add reviewer changes

Alfred-KInyua commented:
hello @m05e5 .
Please have a look at my project, I've incorporated the required changes.

m05e5 commented:
Hi @EasyPanther I'm extremely sorry for the late reply

m05e5 commented:
I don't know if its the same thing you are having but this is how your project looks like
![image](https://user-images.githubusercontent.com/37901469/165648139-c1bfbc59-19ff-4818-9b45-daf9f151791a.png)

m05e5 commented:
The project section is supposed to be in the middle of the intro and the about section

Alfred-KInyua committed:
Add reviewer changes

Alfred-KInyua committed:
Add missing P element in html file

Alfred-KInyua commented:
Thanks, @m05e5  for your review.
I have incorporated the project section as required. Please have a look.

Alfred-KInyua commented:
Hello @m05e5 .
Please review my project.

m05e5 commented:
sure if its ok you'll have to submit on your microverse dashboard for review

m05e5 commented:
Hi i see you've added a button **see project**. Please you are instead supposed to add these cards to your project. In fact you're project is supposed to be kindof the same with the figma design

![image](https://user-images.githubusercontent.com/37901469/165565008-14dd78ef-3190-4bfa-ac35-3ecfd8692cfe.png)

roshan-bajgain commented on best.html:
```diff
@@ -0,0 +1,86 @@
+                </ul>
+            </div>
+        </section>
+        <button type="button">See Projects</button>
+        <section>
+            <H2 class="headingone">About <br>Myself</h2>
+            <p class="paragraph">I
+                Hello I’m a software developer!
+                I can help you build a product ,
+                feature or website Look through some of
+                my work and experience! If you like what
+                you see and have a project you need coded,
+                don’t hestiate to contact me.
+            <p class="lets_connect">LET'S CONNECT</p>
+            <div class="social">
+                <ul class="icons">
+                    <li><img src="images/twitter.png" alt="twitter"></li>
+                    <li><img src="images/linkedin.png" alt="linked in"></li>
+                    <li><img src="images/mid.png" alt="mid"></li>
+                    <li><img src="images/git.svg" alt="git"></li>
+                    <li><img src="images/fin.svg" alt="fing"></li>
+                </ul>
+            </div>
+            <button type="button">Get my resume</button>
+            <section class="lang">
+                <p class="section1"> Languages </p>
+                <p><img src="images/Union.png" alt="drop down"></p>
+            </section>
+            <ul class="languages">
+                <li class="l-items"> <a href="#"><img src="images/js.png" alt="java"><span
```
- [ ]  As part of the main requirements, please add a project section to your portfolio. The project section is an essential part of a portfolio. So kindly implement the suggestion given by the previous reviewer.
- [ ] The middle section of the project is missing so kindly implement it.
## your design
![his](https://user-images.githubusercontent.com/81727988/165733112-709653e7-d808-449e-b502-2033822950c3.png)
## Figma design
![figma](https://user-images.githubusercontent.com/81727988/165733151-85966b52-5a72-4c6e-a959-a7cf84a585dd.png)

Alfred-KInyua replied:
@roshan-bajgain 
should I include the contact section?
Once I finish my work is it possible to communicate with you promptly so that I don't utilize all my reviews, or if I have changes I make them ASAP?. I have used three reviews so far.

roshan-bajgain reviewed (changes requested):
Hi @EasyPanther, 👍  

Great work on making the changes requested by a previous reviewer 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet

<img src="https://media2.giphy.com/media/KG5KpLlr1UEPvwgkiM/giphy.gif"/>

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight :tada: :champagne: 
- Nice PR description and title :+1: 
- Nice README file :heavy_check_mark:
- No linter error :heavy_check_mark:
- GitHub flow is followed :+1:


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Alfred-KInyua committed:
Add reviewers changes

Alfred-KInyua committed:
Solve Linter Errors

EasyPanther commented:
@m05e5 @roshan-bajgain
Please look at my work and advice. I have incorporated all the project changes required.
Please respond as soon as you can. Thanks

Alfred-KInyua committed:
Add missing cards

Alfred-KInyua commented:
@roshan-bajgain please go through my code and if possible, guide me on where I might be wrong.
I have implemented all required sections
.

Alfred-KInyua commented:
@m05e5 @roshan-bajgain.
Thanks for your reviews. I have implemented all changes asked.  

- Formated my "Get resume button" as required. 
- Added missing project sections in my HTML page.
- Added  "**see projects**"  design button. 
- Added the middle part of the Figma template design.

Alfred-KInyua committed:
Add html hover style

roshan-bajgain commented:
@EasyPanther As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project. So I would suggest you to request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Alfred-KInyua committed:
Solve linter Errors

OmarRamoun reviewed (approved):
# 🟢 Project Approved! 🏆

## Hi @EasyPanther 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ implemented previous code review requested changes
- ✔️ good PR communication

## Optional suggestions 🤷‍♂️

You can also consider:

- [ ] [OPTIONAL] Please, always make sure to stick to the design as possible.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️
- This is My **GitHub**: [![@mrramoun](https://img.shields.io/github/followers/MrRamoun?label=Ramoun&style=social)](https://github.com/mrramoun)
- This is My **Twitter**: [![@ramoun16](https://img.shields.io/twitter/follow/ramoun16?label=ramoun16&style=social)](https://twitter.com/ramoun16)
- This is My **Slack**: [![Ramoun](https://img.shields.io/badge/slack-@Ramoun-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
